app/directives/articles/cbd-add-new-article.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+     <a :href="newArticleUrl" :target="target" :class="cssClass">
+        {{buttonText||'Add article'}}
```

Comment:
can use default slot instead
```suggestion
        <slot>'Add article'</slot>
```

in you other code 
<add-article>My button text</add-article>

Reply:
done